janitor/helpers/mlwh_helpers.py

```diff
@@ -77,6 +77,7 @@ def sort_results(
         if (
             result_dict["unordered_barcode"] is None
             and result_dict["ordered_barcode"] is None
+            or result_dict["stored_by"] is None
```

Comment:
When you combine `and` with `or` it's a good idea to use brackets to indicate the order of resolution.  The linter might not let you, but if it will, it can add value to newcomers to your code.

Reply:
I have updated this as per your suggestion to make the conditions and order of resolution clearer.